src/Drawer/index.js

```diff
@@ -151,12 +162,19 @@ const Drawer = ({
   const childrenJSX = (
     <div
       style={depthStyle}
-      className={`drawer drawer--${position} ${
+      className={`drawer drawer-children drawer--${position} ${
```

Comment:
added the `drawer-children` class so we could distinguish the padding applied to the `.drawer.navigation` from the padding applied this container